deploy-scripts/deploy-scripts-constants.ts

```diff
@@ -0,0 +1,19 @@
+import { chainIds } from "../utils/Constants";
+import { TokenDeploymentData } from "./interfaces/deploy-scripts.interface";
+
+const deploymentScriptsFolder = "deploy-scripts";
+
+export const chainsDeployScriptsFolders = {
+  [chainIds.rsk]: [`${deploymentScriptsFolder}/rsk`],
+  [chainIds.mainnetFork]: [`${deploymentScriptsFolder}/rsk`],
+  [chainIds.arbitrumTestnet]: [`${deploymentScriptsFolder}/arbitrum-testnet`]
+};
+
+// Since we're deploying the same tokens on multiple chains, having this data in a constant file will prevent mistakes of typos between chains
+export const tokensDetails: TokenDeploymentData = {
```

Comment:
Exporting from deployment scripts gives us troubles (e.g. when we want to run some experiments and remove deployments scripts). I think we should move it outside.

Reply:
Not sure I follow. Even if you remove deployment scripts, they are in a sub directory for each chain.. so the experiment would be in these sub directories.. where do you see the problem here?

P.S:
This is different than the example you sent me as I create a structure of a main directory, this file and a sub directory for each chain

Reply:
Nevermind, for some reason I was thinking it's a deployment script